src/common/QLogging.cpp

```diff
@@ -17,6 +17,7 @@ Q_LOGGING_CATEGORY(chatterinoCompletion, "chatterino.completion", logThreshold);
 Q_LOGGING_CATEGORY(chatterinoCrashhandler, "chatterino.crashhandler",
                    logThreshold);
 Q_LOGGING_CATEGORY(chatterinoEmoji, "chatterino.emoji", logThreshold);
+Q_LOGGING_CATEGORY(chatterinoEmotes, "chatterino.emotes", logThreshold);
```

Comment:
Random thought unrelated to your change here: It would be nice if emote providers would have their own logging category, and if that didn't require us to define something here